debian/scripts/usr/local/bin/config_save

```diff
@@ -0,0 +1,36 @@
+# Save current configurations to partition containing conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
+
+    # Mount partition as read-write
+    mount /dev/$1 /mnt
+
+    # Find conf.d in root of partition and save current configurations in /etc
+    if [ -d "/mnt/$CONFDIR" ]; then
+        echo Saving configurations to /dev/$1: "/mnt/$CONFDIR/50-node-config-save.tar.gz"
+        # TODO Handle symlinks properly (i.e. excluding them from the tar)
+        tar --create --gzip -f "/mnt/$CONFDIR/50-node-config-save.tar.gz" -C /etc .
+    fi
+
+    # Unmount partition
+    umount /mnt
+}
+
+# Mount proc if not already mounted
+#mount -t proc proc /proc
+
+cat /proc/partitions | while read major minor size name; do
+    # Check each partition matching sd*[0-9] (e.g. sda1) or mmcblk*p* (e.g. mmcblk0p1)
+    case $name in
+        sd*[0-9]|mmcblk*p*)
+            echo Checking for configuration directory on $name
```

Comment:
Right now we only write to partitions with a `conf.d` directory. We should include an empty `conf.d` in the fat16 we create alongside `boot`.

Reply:
+1

Reply:
What is the appropriate place to do this? Is there an obvious common Makefile?